server/routerlicious/.npmrc

```diff
@@ -1,6 +1,6 @@
 engine-strict=true
 strict-peer-dependencies=true
-
+auto-install-peers=true
```

Comment:
Also curious of why this bit.

Reply:
`auto-install-peers=true` this had to be added to the `.npmrc` file in order to successfully install the `pnpm` package. Everything's working seamlessly now even after remvoing the line, but will leave a note if I am able to reproduce the error.